public/app/features/admin/AdminListOrgsPage.tsx

```diff
@@ -16,10 +18,18 @@ const getOrgs = async () => {
   return await getBackendSrv().get('/api/orgs');
 };
 
+const getErrorMessage = (error: any) => {
+  if (error != null && error.data != null && error.data.message != null) {
```

Comment:
```suggestion
  if (error?.data?.message) {
```

Reply:
Or even 
```ts
return error?.data?.message || 'An unexpected error happened.'
```

Reply:
This error message does not provide any useful information to the user.